rush-plugins/rush-migrate-subspace-plugin/src/utilities/project.ts

```diff
@@ -98,3 +98,29 @@ export const getProjectMismatchedDependencies = (projectName: string): string[]
 
   return Array.from(projectMismatches.keys());
 };
+
+export const updateProjectDependency = (
+  projectName: string,
+  dependencyName: string,
+  newVersion: string,
+  rootPath: string = getRootPath()
```

Comment:
Recommend to eliminate optional parameters; generally it is better for the caller to explicitly provide it (and avoid inefficiently calling `getRootPath()` multiple times without realizing it).  Optional parameters are mostly useful for public API's or very heavily used functions, where the convenience of being able to omit a parameter is worth the cost of accidents, or where it's something like an `options` object that can be empty.